Documentation/Tools/HolographicRemoting.md

```diff
@@ -21,6 +23,34 @@ configure the project using the following steps.
 1. Ensure that **Enable MSBuild for Unity** is selected
 1. Click **Apply**
 
+### Debugging HoloLens 2 remoting via Unity package import
```

Comment:
Ah, second thing! It might be helpful to have text (i.e. error messages, error logs) that can help a person understand what errors they're seeing - i.e. how do I know that I'm hitting a DotNetWinRt NuGet package resolution issue?

Reply:
Sounds good! I can add some images too